src/index.ts

```diff
@@ -258,4 +259,36 @@ program
     }
   );
 
+program
+  .command("delete")
+  .argument("[projectId]", "ID of the project you want to delete.")
+  .argument("[-f]", "Skip confirmation prompt for deletion.")
+  .description("Delete the project described by the provided ID. If no ID is provided, lists all the projects and IDs.")
+  .action(async (projectId  = "", forced  = false) => {
+    // check if user is logged in
+    const authToken = await readToken().catch(() => undefined);
+
+    if (!authToken) {
+      console.log(
+        "You are not logged in. Run 'genezio login' before you delete your function."
+      );
+      exit(1);
+    }
+
+    const result = await deleteProjectHandler(projectId, forced).catch((error: AxiosError) => {
```

Comment:
If I write this command:

```
genezio delete projectId b810a15c-cf7c-4aa6-a386-7091d4e44129
```

The value of the projectId variable is "projectId" which causes a 500.